app/controllers/OAuth.scala

```diff
@@ -1,23 +1,22 @@
 package controllers
 
-import javax.inject.Inject
-
+import actions.{CommonActions, OAuthActions}
 import com.gu.googleauth.GoogleAuthFilters.LOGIN_ORIGIN_KEY
- import actions.OAuthActions
 import com.gu.googleauth.{GoogleAuth, UserIdentity}
 import configuration.Config
 import model.FlashMessage
-import play.api.Play.current
-import play.api.libs.json.Json
-import play.api.mvc._
 import play.api.libs.concurrent.Execution.Implicits.defaultContext
+import play.api.libs.json.Json
 import play.api.libs.ws.WSClient
+import play.api.mvc._
 
 import scala.concurrent.Future
 
-class OAuth @Inject()(val wsClient: WSClient) extends Controller with OAuthActions{
-  val ANTI_FORGERY_KEY = "antiForgeryToken"
-  implicit val iWsClient = wsClient
+class OAuth (val wsClient: WSClient, commonActions: CommonActions, oAuthActions: OAuthActions) extends Controller {
+
+  import commonActions._
```

Comment:
this stuff is removed due to the play anti forgery changes in https://github.com/guardian/play-googleauth/pull/52 - I tested resubmitting with the same key over a minute after the origial request and it kicked it out with an error.  The error was a little more revealing than I'd like, but it was an out of date token, so it was working in some way.

Reply:
👍